papers/oleksandr_pavlyk/fft_optimizations/fft.rst

```diff
@@ -0,0 +1,92 @@
+   +----------------+-----------------+-----------------+-----------------+-----------------+-------------------+------------------+
+   | `arg.shape`    | `(3*10**6,)`    | `(3*10**6,)`    | `(1860, 1420)`  | `(1860, 1420)`  | `(273, 274, 275)` | `(273, 274, 275)`|
+   +-------------+--+-----------------+-----------------+-----------------+-----------------+-------------------+------------------+
+   |             |cd| 1.40 |+-| 0.02  | 0.885 |+-| 0.005| 0.090 |+-| 0.001| 0.067 |+-| 0.001| 0.868 |+-| 0.007  | 0.761 |+-| 0.001 |
+   | IDP 2017.0.3+--+-----------------+-----------------+-----------------+-----------------+-------------------+------------------+
+   |             |cs| 0.734 |+-| 0.004| 0.450 |+-| 0.002| 0.056 |+-| 0.001|0.041 |+-| 0.0002| 0.326 |+-| 0.003  | 0.285 |+-| 0.002 |
+   +-------------+--+-----------------+-----------------+-----------------+-----------------+-------------------+------------------+
+   |             |cd| 1.77 |+-| 0.02  | 1.760 |+-| 0.012| 2.208 |+-| 0.004| 2.219 |+-| 0.002| 22.77 |+-| 0.38   | 22.7  |+-| 0.5   |
+   | IDP 2017.0.1+--+-----------------+-----------------+-----------------+-----------------+-------------------+------------------+
+   |             |cs| 5.79 |+-| 0.14  | 5.75 |+-| 0.02  | 1.996 |+-| 0.1  | 2.258 |+-| 0.001| 27.12 |+-| 0.05   | 26.8  |+-| 0.25  |
+   +-------------+--+-----------------+-----------------+-----------------+-----------------+-------------------+------------------+
+   |             |cd| 26.06 |+-| 0.01 | 23.51 |+-| 0.01 | 4.786 |+-| 0.002| 3.800 |+-| 0.003| 67.69 |+-| 0.12   | 81.46 |+-| 0.01  |
+   | pip numpy   +--+-----------------+-----------------+-----------------+-----------------+-------------------+------------------+
+   |             |cs| 28.4 |+-| 0.1   | 11.9 |+-| 0.05  | 5.010 |+-| 0.003| 3.77  |+-| 0.02 | 69.49 |+-| 0.02   | 80.54 |+-| 0.07  |
+   +-------------+--+-----------------+-----------------+-----------------+-----------------+-------------------+------------------+
+
+
+Direct support for multivariate transforms along distinct array axis. Even when multivariate transform ends up being computed as iterations
+of one-dimensional transforms, all subsequent iterations are performed in place for efficiency.
+
+The update also provides dedicated support for complex FFTs on real inputs, such as `np.fft.fft(real_array)`, by leveraging corresponding
+functionality in MKL [2]_.
+
+.. Illustrate the point that this became faster
+
+Dedicated support for specialized real FFTs, which only store independent complex harmonics. Both `numpy.fft.rfft` and `scipy.fftpack.rfft`
+storage  modes are natively supported via Intel MKL.
+
+.. show rfft is faster in update 2 relative to update 1
+
+
+
+References
+----------
+
+
+.. |C| unicode:: 0xA9 .. copyright sign
+   :ltrim:
+.. |R| unicode:: 0xAE .. registered sign
+   :ltrim:
+.. |TM| unicode:: 0x2122 .. trade mark sign
+   :ltrim:
+.. |+-| unicode:: 0x00B1 .. plus-minus sign
+   :ltrim:
```

Comment:
this is already defined in the main body of the paper. It might be also better to use standard definitions instead (can do later)